solana/ts/lib/ntt.ts

```diff
@@ -259,12 +318,10 @@ export class NTT {
     };
     const pubkeys = collectPubkeys(entries).map((pk) => pk.toBase58());
 
-    var existingLut: web3.AddressLookupTableAccount | null = null;
+    let existingLut: web3.AddressLookupTableAccount | null = null;
     try {
-      existingLut = await this.getAddressLookupTable(false);
-    } catch {
-      // swallow errors here, it just means that lut doesn't exist
-    }
+      existingLut = await getAddressLookupTable(program, pdas);
+    } catch {}
```

Comment:
this catchall situation makes me a little uneasy -- maybe we should just change `getAddressLookupTable` to return `web3.AddressLookupTableAccount | null` (so genuine exceptions are propataged)